app/api/chat/conversation/[slug]/message/route.ts

```
@@ -1,5 +1,5 @@
 import { and, eq } from "drizzle-orm";
-import { createMessageBodySchema } from "@helperai/client";
+import { createMessageBodySchema, type ToolRequestBody } from "@helperai/client";
```

ESLint

[error] 'ToolRequestBody' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)